### PR TITLE
Improvements: Changed the Color of the text 'Code' in the footer

### PR DESCRIPTION
**Pull request by Paul-Annay**
This PR fixes the color of the text in the footer to match according to the color of the header.

Fixes: #74

### CODE REVIEW

**Comment by AdarshRawat1** [comment]
@Paul-Annay Include the screenshots of changes.

File: index.html [hunk 1]
```diff
@@ -180,7 +186,9 @@ <h2>ML-X</h2>
       <div class="view_more__projects">
         <a href="https://github.com/orgs/OpenCodeEra/repositories" target="_blank">View More</a>
         <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor">
-          <path fill-rule="evenodd" d="M12.293 5.293a1 1 0 011.414 0l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414-1.414L14.586 11H3a1 1 0 110-2h11.586l-2.293-2.293a1 1 0 010-1.414z" clip-rule="evenodd" />
+          <path fill-rule="evenodd"
```
Comment (AdarshRawat1) [inline index.html:189]: Avoid formatting the code that you are not working on, this way we can track the changes better.

**Comment by AdarshRawat1** [comment]
Merge conflicts in PRs can occur due to differences in code formatting, especially when different contributors use different formatting styles. 

However, they can also occur when multiple developers make conflicting changes to the same part of the codebase.

File: index.html (hunk 1 above)
Comment (Paul-Annay) [reply index.html:189]: My apologies, I was using an extension which auto-formats on file save therefore I didn't notice the changes. Should I make any further modifications?

**Pull request merged**